tests/testcases/030_check-network.yml

```diff
@@ -12,19 +12,25 @@
       bin_dir: "/usr/local/bin"
     when: not ansible_os_family in ["CoreOS", "Container Linux by CoreOS"]
 
+  - name: Check kubectl output
+    shell: "{{bin_dir}}/kubectl get pods --all-namespaces -owide"
+    register: get_pods
+
+  - debug: msg="{{get_pods.stdout}}"
```

Comment:
Ah ok, I'm fine with leaving it then.

Btw, I use human_log.py for local debugging as it prints all the output from all commands in a readable form. Maybe that's an option for CI as well?

Can be found here: https://gist.github.com/dmsimard/cd706de198c85a8255f6